service/src/test/java/uk/gov/hmcts/reform/fpl/service/additionalapplications/UploadAdditionalApplicationsServiceTest.java

```diff
@@ -137,8 +129,6 @@ void shouldBuildExpectedC2DocumentBundle() {
         assertThat(actual.getC2DocumentBundle().getApplicantName()).isEqualTo(APPLICANT_NAME);
 
         assertC2DocumentBundle(actual.getC2DocumentBundle(), supplement, supportingEvidenceBundle);
-
-        verifyNoInteractions(conversionService);
```

Comment:
can you change this assertion to verify if documentConversionService.convertToPdf() is invoked?

Reply:
Done